### DISCUSSION TITLE
Mergify failing to detect status on GitHub LGTM jobs for Dependabot PRs

### DISCUSSION THREAD

**Discussion by rbclark** [Mergifyio/mergify#2542] [Q&A]
I am running into an interesting issue while in the process of migrating from Dependabot auto-merging PRs over to having mergify perform the merging. For context, I have some required checks setup that must pass before PRs can be merged. One of those checks is LGTM, which seems to be not running on Dependabot PRs since Dependabot changes do not change any of the code, just dependency versions:

![Screen Shot 2021-05-05 at 4 11 19 PM](https://user-images.githubusercontent.com/2308869/117203025-89a9de00-adbc-11eb-9ca6-c9997700893e.png)
(note the square instead of a green check, it seems Github refers to this is a neutral check).

Unfortunately, it seems mergify is not detecting these neutral checks and is instead only looking for 'passed' checks, so I am seeing this on Mergify for the same PR:

![Screen Shot 2021-05-05 at 4 12 49 PM](https://user-images.githubusercontent.com/2308869/117203201-beb63080-adbc-11eb-9f93-40fa78249d8d.png)

My actual mergify config doesn't even mention GitHub Branch Protection checks, this is something that mergify waits for automatically. (Configuration can be found [here](https://github.com/mitre/heimdall2/blob/master/.github/mergify.yml))

Could someone provide some guidance on this?

**Comment by jd** [comment]
Hi @rbclark!

Congratulations, you found a limitation in Mergify. 😉 
The good news is that @sileht already accepted the challenge and started a pull request to implement what was missing: https://github.com/Mergifyio/mergify-engine/pull/2544

🤞 that should be merged in the next hours and should solve your issue!

**Reply by rbclark** [reply]
Yep everything is working as expected now, thanks!
